train_model.py

```diff
@@ -350,11 +378,12 @@ def get_params(self) -> dict:
         model.train()
         train_total_loss = 0
         train_total_samples = 0
-        for batch_idx, (inputs, targets) in enumerate(tqdm(train_loader,
-                                                           desc="Train")):
+        for batch_idx, batch_seqs in enumerate(tqdm(
```

Comment:
Maybe call this train_batch_seqs to follow pattern of val_batch_seqs below. However, not required. Can be discussed.